scrapy/core/http2/stream.py

```diff
@@ -16,6 +16,7 @@
 from scrapy.http import Request
 from scrapy.http.headers import Headers
 from scrapy.responsetypes import responsetypes
+from scrapy.utils.python import to_unicode
```

Comment:
`str` should work, see https://github.com/scrapy/scrapy/issues/4547

Reply:
Makes sense to stop using `to_unicode`, but please be [mindful of the encoding](https://github.com/scrapy/scrapy/pull/4661#issuecomment-654901531). It might not be an issue in these cases, but it's something to think about IMHO.